### ISSUE TITLE
cannot resolve module `fs` in jade

### ISSUE THREAD

**Issue by rbalicki2** [webpack/analyse#24]
I am getting the following error when I run `grunt`, `grunt dev` or `grunt deploy`:

```
ERROR in ./~/jade/lib/runtime.js
Module not found: Error: Cannot resolve module 'fs' in /Users/robertbalicki/Documents/code/analyse/node_modules/jade/lib
 @ ./~/jade/lib/runtime.js 155:12-25
```

I can fix it (hackily?) by installing `browserify-fs` as `fs` in either `node_modules` or `web_modules`. It has some unmet peer dependencies, which I haven't dug into.

Please advise - happy to fix this properly, but I haven't grokked the library fully enough to know what direction to go in.

FWIW, setting `target: node` on the webpack config breaks the `require.include(sigma.js)` line, but seems to (obviously) fix the `fs` error.

**Comment by kirbysayshi** [comment]
This is fixed by #23, at least locally. Public website appears to be broken at the moment.

**Comment by a-r-d** [comment]
Just merged this which fixes the issue:
https://github.com/webpack/analyse/pull/23
